src/core/utils/index.js

```diff
@@ -224,6 +224,16 @@ var utils = module.exports = {
     },
 
     /**
+     * Retunrs sign of number
+     *
+     * @param n {number}
+     * @returns {number} 0 if n is 0, -1 if n is negative, 1 if n i positive
+     */
+    sgn: function (n){
+        return n === 0 ? 0 : n < 0 ? -1 : 1;
```

Comment:
@adelciotto Hi. I will use implementation you send, because it's faster then mine. And about the name - I can change it, but [```sgn```](https://en.wikipedia.org/wiki/Sign_function) is widely used in mathematics, that's why I used it. And it's abbreviation of 'signum' not 'sign', btw :) . 
But on the other hand - there is [```Math.sign```](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sign) in ```ES6```, so developers might be more familiar with ```sign```.  I'm not sure - I guess I'll wait a bit with this decision, maybe more people will express their opinion about the name.

@englercj  Was that :+1: for name change or function itself ? :)